src/state.txt

```diff
@@ -0,0 +1 @@
+[{"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.0], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}, {"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}, {"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}, {"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}, {"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}, {"OBSTACLES": [0.8, 0.2, 0.2, 1.0], "OBSTACLES_TRI_WF": [1.0, 1.0, 1.0, 1.0], "OBSTACLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_TRI_WF": [1.0, 1.0, 1.0, 0.5], "TRAVERSABLES_SQR_WF": [1.0, 1.0, 1.0, 0.5], "START": [0.5, 0.0, 0.5, 1.0], "GOAL": [0.0, 1.0, 0.0, 0.5], "PATH": [1.0, 1.0, 1.0, 0.5], "FRINGE": [0.2, 0.2, 0.8, 1.0], "EXPLORED": [0.4, 0.4, 0.4, 1.0]}]
```

Comment:
Remove this file, it shouldn't be tracked (.gitignore should already be handling this)